### DISCUSSION TITLE
Error while  creating  VPC using Terraform

### DISCUSSION THREAD

**Discussion by Maheshdvp** [learndevopsonline/learndevopsonline#1660] [Q&A] [answered]
Hi team, I could'nt able to understand this error. could you please explain error and solution to solve this problem.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/f1c72ebe-8950-45c7-8923-dee90184aa8e)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/7434fa6e-31c1-4f9b-a17f-223ae48e7b87)

**Comment by sreedharm07** [comment]
@Maheshdvp 
its seems like you created bucket with some private permissions 
please check that 
or check that bucket exists and also 
check role that attached to the workstation has access to s3 bucket

**Reply by Maheshdvp** [reply]
No IAM role is attached is that the issue
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/5fe9698c-4210-4894-af57-a9b2df5e12b4)

**Reply by sreedharm07** [reply]
yes,
 please attach role to workstation

**Reply by Maheshdvp** [reply]
Yes its working fine but showing another error like this but my code seems to be fine
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/051efe00-da08-422f-a6fb-7b850e083747)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/485891cb-9a0f-4a05-abe8-5483f8b7b760)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/4590c0c8-4dd4-41de-810c-d653df976f43)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/17ad3446-4c3f-4c86-88e9-05295e8b7c45)

**Reply by sreedharm07** [reply]
in this file remove double quoets for var.cidr

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/8198c62a-fa00-4729-9561-e61d254c2877)

**Reply by Maheshdvp** [reply]
Thanks @sreedharm07 done with changes you had told but here comes an another error I had double checked video too 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/ec37acee-c85b-4be2-aa54-3e22cbb8ac1d)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/6a7a11ab-c955-42a9-945e-d705cf8b1f1d)

**Reply by sreedharm07** [reply, marked as answer]
check iam policy 
its telling that there is no policy to create vpc 
did you given admin policy to workstation ?

**Reply by Maheshdvp** [reply]
No did'nt given

**Reply by sreedharm07** [reply]
Give admin access policy to role and attach that role to workstation

**Reply by Maheshdvp** [reply]
Just attached full admin access it was working fine
